lib/stripped-build-plugins.js

```diff
@@ -39,7 +39,20 @@ module.exports = function(environment) {
 
   if (process.env.INSTRUMENT_HEIMDALL === 'false') {
     plugins.push(stripHeimdall);
-    uniqueAdd(filteredImports, 'ember-data/-private/debug', ['instrument']);
+    /**
+     * The original line used the package name.  This was changed to remove the
+     * instrument paths from adapters since it changed the test behavior.
+     *
+     * This should not be considered worked!  There are other places that import
+     * this from a different relative path. Its also a refactoring hazard because any
+     * new imports wouldn't be removed.
+     *
+     * Not sure what the solution should be; update the path matching for filteredImport
+     * to use regex?
+     *
+     * original: uniqueAdd(filteredImports, 'ember-data/-private/debug', ['instrument']);
+     */
+    uniqueAdd(filteredImports, '../-private/debug', ['instrument']);
```

Comment:
Is it a bad idea to support regex, or somehow modify the stripped-build-plugin to be more flexible?

Another option could be to have an explicit name for the global/package-level debug vs the -private debug.  I think it would be confusing to have both `ember-data/-private/debug` and `../debug` (for system/debug) imports in the code.  This might actually be a larger discussion about having the same (or similar) name at different levels (ex. addon/serializer, addon/serializers, addon/-private/serializers). 

So what should the path forward be? To remove the relative path for references to addon/-private/debug?